GUI/MainWindow.xaml.cs

```diff
@@ -5,24 +5,319 @@
+        {
+
+        }
+
+        private void BtnAddFastq2Proteins_Click(object sender, RoutedEventArgs e)
+        {
+            var dialog = new Fastq2ProteinsWFWindows();
+            if (dialog.ShowDialog()==true)
+            {
+                AddTaskToCollection(dialog.TheTask);
+                UpdateTaskGuiStuff();
+            }
+        }
+
+        private void BtnAddLncRNADiscover_Click(object sender, RoutedEventArgs e)
+        {
+            var dialog = new LncRNADiscoverWFWindows();
+            if (dialog.ShowDialog() == true)
+            {
+                AddTaskToCollection(dialog.TheTask);
+                UpdateTaskGuiStuff();
+            }
+        }
+
+        private void UpdateTaskGuiStuff()
+        {
+            if (staticTasksObservableCollection.Count == 0)
+            {
+                RunWorkflowButton.IsEnabled = false;
+                RemoveLastTaskButton.IsEnabled = false;
+                ClearTasksButton.IsEnabled = false;
+                ResetTasksButton.IsEnabled = false;
+            }
+            else
+            {
+                RunWorkflowButton.IsEnabled = true;
+                RemoveLastTaskButton.IsEnabled = true;
+                ClearTasksButton.IsEnabled = true;
+            }
+        }
+
+        private void RunWorkflowButton_Click(object sender, RoutedEventArgs e)
+        {
+            dynamicTasksObservableCollection = new ObservableCollection<InRunTask>();
+            for (int i = 0; i < staticTasksObservableCollection.Count; i++)
+            {
+                dynamicTasksObservableCollection.Add(new InRunTask("Workflow" + (i + 1) + "-" + staticTasksObservableCollection[i].spritzWorkflow.WorkflowType.ToString(), staticTasksObservableCollection[i].spritzWorkflow));
+            }
+            workflowTreeView.DataContext = dynamicTasksObservableCollection;
+            EverythingRunnerEngine a = new EverythingRunnerEngine(dynamicTasksObservableCollection.Select(b => new Tuple<string, SpritzWorkflow>(b.DisplayName, b.workflow)).ToList(), rnaSeqFastqCollection.Where(b => b.Use).Select(b => b.FilePath).ToList(), genomeFastaCollection.Where(b => b.Use).Select(b => b.FilePath).ToList(), geneSetCollection.Where(b => b.Use).Select(b => b.FilePath).ToList(), OutputFolderTextBox.Text);
+            var t = new Task(a.Run);
+            t.Start();
+        }
+
+        private void DataGridCell_MouseDoubleClick(object sender, MouseButtonEventArgs e)
+        {
+            var ye = sender as DataGridCell;
+            if (ye.Content is TextBlock hm && !string.IsNullOrEmpty(hm.Text))
+            {
+                System.Diagnostics.Process.Start(hm.Text);
+            }
+        }
+
+        private void BtnAddGenomeFASTA_Click(object sender, RoutedEventArgs e)
+        {
+            Microsoft.Win32.OpenFileDialog openPicker = new Microsoft.Win32.OpenFileDialog()
+            {
+                Filter = "Database Files|*.fa",
```

Comment:
Sometimes these are spelled out as *.fasta -- could we represent that here?

Reply:
Looks like you can do it with `Filter = "Genome Fasta Files (*.fa; *.fasta)|*.fa; *.fasta",`

Oh, could you also make this clearer that it's supposed to be a genome fasta, so someone doesn't try to drop in a protein fasta file? I guess we should eventually check that it's really a genome fasta... I made an issue for that.